docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -74,6 +74,19 @@ relative to when data was written last to a rolled over index.
 The previous phase's actions must complete before {ILM} will check `min_age` and
 transition into the next phase.
 
+=== Phase Execution
+
+beta[]
+
+The current phase definition, of an index's policy being executed, is stored
+in the index's metadata. The phase and its actions are compiled into a series
+of discrete steps that are executed. Since some {ILM} actions are more complex
+and involve multiple actions against an index, each of these are done in their
+own step. This provides better visibility into the execution of the phases. The
```

Comment:
```suggestion
done in isolation in a unit called a step. This provides better visibility into
the execution of the phases. The
```

Reply:
continuation of suggestion form above since Github's suggest feature only allows you to provide single line suggestions